apps/admin-gui/src/app/shared/components/two-entity-attribute-page/two-entity-attribute-page.component.ts

```diff
@@ -43,32 +47,98 @@ export class TwoEntityAttributePageComponent implements OnInit, OnChanges {
   @Input()
   secondEntity: string;
 
-  @Input()
-  entityValues: Resource[] | Facility[] | Group[];
-
+  entityValues: Resource[] | Facility[] | Group[] | RichMember[] | User[] = [];
   attributes: Attribute[] = [];
   selection = new SelectionModel<Attribute>(true, []);
+  specificSecondEntity: Resource | Facility | Group | RichMember | User;
 
-  filter = '';
-  filteredEntityValues: Resource[] | Facility [] = [];
+  loading = false;
+  innerLoading = false;
 
-  loading: boolean;
-  innerLoading: boolean;
-
-  filterMessage: string;
   noEntityMessage: string;
-  noFilteredEntityMessage: string;
 
   ngOnInit(): void {
-    if (this.firstEntity === 'member') {
-      this.setMessages(this.secondEntity.toUpperCase());
-    } else if (this.firstEntity === 'group') {
-      this.setMessages('RESOURCE');
-    } else (this.setMessages(this.firstEntity.toUpperCase()));
+    this.loadEntityValues();
+    this.setMessages(this.secondEntity.toLowerCase());
   }
 
-  ngOnChanges(): void {
-    this.filteredEntityValues = this.entityValues;
+  loadEntityValues() {
```

Comment:
in whole this method you are calling method like getAllowedResources but you are not checking if the user have priviledge to call this method, how it is right now this can very easily happen
![image](https://user-images.githubusercontent.com/32871167/98958001-1d7fa900-2502-11eb-92d8-acb8aee1e42a.png)
If the user not to authorized to call the method which will show him the select where he can select member/resource/.. then I wouldnt show him the tab actually